Assets/HotmaxEditorVR/Hittables/deletePanel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+
+public class deletePanel : MonoBehaviour, IHittable
+{
+    //--local refs
+    private stateManager _stateManagerMutatorRef;
+    private GameObject _selectedObject;
+
+    public Transform target;
+
+    private void Awake()
+    {
+        _stateManagerMutatorRef = GameObject.FindObjectOfType(typeof(stateManager)) as stateManager;
+        stateManager.selectedObjectEvent += updateSelectedObject;
+    }
+
+    void Update()
+    {
+        transform.LookAt(target);
+    }
+
+    private void OnApplicationQuit()
```

Comment:
can you group the awake and onApplication quit functions together, and put update underneath

Reply:
got it